packages/shared/src/components/cards/ActionsButtons/UpvoteButtonIcon.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useRef, useState } from 'react';
+import classNames from 'classnames';
+import { withExperiment } from '../../withExperiment';
+import { feature } from '../../../lib/featureManagement';
+import { UpvoteIcon } from '../../icons';
+import { IconProps, IconSize } from '../../Icon';
+import { userPrefersReducedMotions } from '../../../styles/media';
+import { useConditionalFeature, useMedia } from '../../../hooks';
+import styles from './ActionButtons.module.css';
+
+function useAnimatedActionButtons(options: { isUpvoteActive: boolean }) {
+  const { isUpvoteActive } = options;
+  const initialUpvote = useRef(isUpvoteActive);
+  const [userClicked, setUserClickedUpvote] = useState(false);
+
+  const haveUserPrefersReducedMotions = useMedia(
+    [userPrefersReducedMotions.replace('@media', '')],
+    [false],
+    false,
+    false,
+  );
+  const currentVersion = useConditionalFeature({
+    feature: feature.animatedUpvote,
+    shouldEvaluate: !haveUserPrefersReducedMotions,
+  });
+  const isAnimatedVersion = !!currentVersion?.value;
```

Comment:
I'm not sure you need it here. I think basically the evaluation can move to withExperiment right @capJavert ?

Reply:
It is still mandatory because of the `reduces motion` I didn't find a way to insert this in the `withExperiment ` logic  :(

Reply:
Ah yeah was wondering if we could put that inside the `value` evaluation there?
(Ante might know best since he made it)

Reply:
Trying to implement it 💪

Reply:
Now it should works without the `useConditionalFeature ` :)